test/ibmq/test_ibmq_job.py

```diff
@@ -78,15 +84,15 @@ def test_run_device(self, backend):
 
     def test_run_multiple_simulator(self):
         """Test running multiple jobs in a simulator."""
-        num_qubits = 16
+        num_qubits = 8
         qr = QuantumRegister(num_qubits, 'qr')
         cr = ClassicalRegister(num_qubits, 'cr')
         qc = QuantumCircuit(qr, cr)
         for i in range(num_qubits - 1):
             qc.cx(qr[i], qr[i + 1])
         qc.measure(qr, cr)
-        num_jobs = 5
-        job_array = [self.sim_backend.run(transpile([qc] * 20), shots=2048)
+        num_jobs = 3
```

Comment:
Since we shouldn't run into any job limit issue (because this test uses a non-open provider), and the purpose of this test is to test "running multiple jobs [in parallel]", I don't see why we need to reduce the number of jobs.

Reply:
This is just to accomplish the same thing using less CPU, if possible. I don't think this reduction should introduce any breaking change though?